ballista/core/src/lib.rs

```diff
@@ -29,6 +29,7 @@ pub mod event_loop;
 pub mod execution_plans;
 /// some plugins
 pub mod plugin;
+pub mod test_util;
```

Comment:
It seems questionable to export test dependencies as part of the main artifact. Perhaps a `#[cfg(test)]` would resolve that?

Reply:
thanks, i moved test_util to ballista-examples package.